translations/es/translation.json

```diff
@@ -1,18 +1,18 @@
 {
   "fes": {
-    "autoplay": "Su browser impidío un medio tocar (de '{{src}}'), posiblemente porque las reglas de autoplay. Para aprender más, visite {{link}}.",
-    "checkUserDefinedFns": "",
+    "autoplay": "Su browser impidío un medio tocar (de '{{src}}'), posiblemente por las reglas de autoplay. Para aprender más, visite {{link}}.",
```

Comment:
The original is "The media that tried to play (with '{{src}}') wasn't allowed to by this browser, most likely due to the browser's autoplay policy.\n\n+ More info: {{url}}"

I suggest "navegador" instead of browser. Also the grammatic of the first sentence would be better in my opinion as:  "Su navegador impidió la reproducción del contenido multimedia (de '{{src}}'.), posiblemente debido a las reglas de autoplay..."